packages/teams-js/src/public/interfaces.ts

```diff
@@ -758,6 +758,14 @@ export interface Context {
    * They help pre-fill the dialog with necessary information (`dialogParameters`) along with other details.
    */
   dialogParameters?: Record<string, string>;
+
+  /**
+   * @deprecated
+   * As of 2.0.0, please use {@link app.AppInfo.appId | app.Context.app.appId} instead
+   *
+   * App id that is used by Hosts to distinguish between different apps sideloaded or in store
+   */
+  appId: string;
```

Comment:
Since this is a new feature, can I know why we need to add this parameter to a deprecated interface and as a deprecated element?

Reply:
I believe the Context deprecated interface is still used in the legacyContext transformation so I kept it

Reply:
This one should probably also be optional